seeds/data/project-versions.json

```diff
@@ -392,7 +392,61 @@
         }
       ]
     },
-    "status": "draft"
+    "status": "draft",
+    "createdAt": "2020-06-30",
+    "updatedAt": "2019-06-30"
+  },
+  {
+    "projectId": "04956663-711b-4bda-b201-46d9eeabf9d4",
+    "data": {
+      "title": "Project with deleted protocols",
+      "species": ["mice", "rats"],
+      "protocols": [
+        {
+          "id": "43f5e295-277c-4e60-9759-e6c3612d6bfe",
+          "steps": [],
+          "title": "First protocol",
+          "species": [ "mice" ],
+          "conditions": [],
+          "speciesDetails": [
+            {
+              "id": "39415b58-73bc-4040-b7dd-d5bef57de5f2",
+              "name": "Mice",
+              "reuse": false,
+              "value": "mice",
+              "life-stages": [ "adult" ],
+              "continued-use": false,
+              "maximum-animals": "100",
+              "maximum-times-used": "100"
+            }
+          ]
+        },
+        {
+          "id": "997659c6-6789-49db-a5cd-1cfcb6856406",
+          "steps": [],
+          "title": "Deleted protocol",
+          "deleted": false,
```

Comment:
you've got `deleted` twice?

Reply:
I just copied the version that was already there. I've fixed both.